app/views/owners/index.html.erb

```diff
@@ -0,0 +1,20 @@
+
+
+
+<div class="row">
+    <div class="col-md-6 col-md-offset-1">
+        <!-- <= render 'form' %> -->
+        <h1>The Owner List (<%= Owner.count %>)</h1>
```

Comment:
You can remove the extra database query by using `@owner.count` here instead of `Owner.count`.

Reply:
Oh, that makes sense! I'll change it to `@owners.count`